taccsite_cms/static/site_cms/css/src/_imports/_tests/postcss-preset-env.css

```diff
@@ -26,6 +26,26 @@ Styleguide _Test.PresetEnv
   }
 }
 
+/*! Goal: */
+@media (max-width:60em){._test{background-color:red}}@media (max-width:60em){._test{background-color:blue}}
```

Comment:
I had hoped that I could achieve `@media (max-width:`__`30em`__`){._test{background-color:red}}@media (max-width:`__`60em`__`){._test{background-color:blue}}`, but this proves that I cannot.